### PR TITLE
Add sort time entries by client functionality

### PR DESCRIPTION
**Pull request by HermanVulkers**
Hey guys, hier zitten oude commits bij. Alleen de commits vanaf vrijdag 4 maart hoeven gereviewed te worden.

### CODE REVIEW

**Review by RubenWerdmuller — DISMISSED** [review]

File: db.json [hunk 1]
```diff
@@ -29,11 +22,86 @@
...
+      "name": "Hike One"
+    },
+    {
+      "id": 3,
+      "name": "Nationale Nederlanden"
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (hans-acc) [inline db.json:107]: add newline

File: pages/team-members.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          <PageContainer>
+            <TeamMembers initialTeamMembers={initialTeamMembers} />
+          </PageContainer>
+        </StoreProvider>
+      </ThemeProvider>
+    </>
+  );
+};
+
+export default Homepage;
```
Comment (hans-acc) [inline pages/team-members.tsx:52]: add newline

File: pages/team-members.tsx [hunk 3]
```diff
@@ -2,14 +2,17 @@ import { ThemeProvider } from "styled-components";
 
 import { Header } from "../src/components/header/Header";
 import { PageContainer } from "../src/components/shared/PageContainer";
+import { TeamMembers } from "../src/components/team-members/TeamMembers";
```
Comment (hans-acc) [inline pages/team-members.tsx:6]: cleanup the whitelines svp :)

File: src/components/header/Header.styled.ts [hunk 4]
```diff
@@ -9,13 +9,15 @@ export const Header = styled.header<{ isMenuVisible: boolean }>`
   justify-content: start;
   position: fixed;
   width: 100%;
+  z-index: 2;
```
Comment (hans-acc) [inline src/components/header/Header.styled.ts:12]: z-index is not the nicest way to solve these kinds of issues. maybe there is another solution

File: src/components/shared/TimeEntryHeader.tsx [hunk 5]
```diff
@@ -1,20 +1,50 @@
+import React, { useContext } from "react";
+
 import * as Styled from "../time-entries/TimeEntries.styled";
```
Comment (hans-acc) [inline src/components/shared/TimeEntryHeader.tsx:4]: remove whitelines

File: src/types/TeamMember.types.ts [hunk 6]
```diff
@@ -1,11 +1,12 @@
 export interface TeamMemberProps {
   emailAddress: string;
-  employer: string;
+  client: string;
```
Comment (hans-acc) [inline src/types/TeamMember.types.ts:3]: alphabetical order:)

**Review by hans-acc — APPROVED** [review]
approved for now. very niceee. minor pointers you need to fix before merging 🙈

**Review by nathaliebru — APPROVED** [review]
Looks nice!

**Pull request merged**